index.html

```diff
@@ -33,19 +33,17 @@
 		        </label>
 
 		        <ul class="menu">
-		        	<div class="menu__wrapper">
+		        		<li><a class="menu__item" href="#">Home</a></li>
 		    			<li><a class="menu__item" href="#">Destinations</a></li>
 		            	<li><a class="menu__item" href="#">About</a></li>
 		            	<li><a class="menu__item" href="#">Partner</a></li>
-		        	</div>
-		        	<div class="menu__buttons">
+
 		        		<button class="login">Login</button>
 						<button class="register">Register</button>	
-		        	</div>
 		        </ul>
```

Comment:
you need to delete it at all

you have your `<nav class="nav"> ... </nav> `  and if screen-width less then 768px you need to change styles of `.nav`